pkg/policies/policies.go

```diff
@@ -17,13 +17,19 @@
 package policies
 
 import (
+	"github.com/ossf/allstar/pkg/policies/binary"
 	"github.com/ossf/allstar/pkg/policies/branch"
+	"github.com/ossf/allstar/pkg/policies/outside"
+	"github.com/ossf/allstar/pkg/policies/security"
 	"github.com/ossf/allstar/pkg/policydef"
 )
 
 // GetPolicies returns a slice of all policies in Allstar.
 func GetPolicies() []policydef.Policy {
```

Comment:
Not needed in this PR, but might be useful to consider registering policies like this: https://github.com/ossf/scorecard/blob/35267c251496135a0a5e9097470583f6c447b68c/checks/all_checks.go#L22

In case users want to run only a smaller list of policies it'll be useful.

Reply:
Yes, I'm familiar with this pattern. While useful, I think it reduces clarity, so I'd only want to use if with good reason. At the user level: which policies are run is set by configuration. If we do get more app operators that wish to exclude policies at compile time it would make sense.